go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20171020084629-8d44bfdf1030
 	github.com/pingcap/kvproto v0.0.0-20190826051950-fc8799546726
 	github.com/pingcap/log v0.0.0-20190307075452-bd41d9273596
-	github.com/pingcap/parser v0.0.0-20190910040957-e998b3c52469
+	github.com/pingcap/parser v0.0.0-20191010030522-6e4f15d4819b
```

Comment:
Update the parser's commit?
Seems that it's the commit hash of that pr instead of the release2.1 branch's commit hash.

Reply:
Updated. PTAL again, thanks.